jekyll/_cci2/migrating-from-1-2.md

```diff
@@ -181,29 +180,28 @@ jobs:
   build:
     working_directory: /root/my-project
     docker:
-      - image: ruby:2.3
 
-      # You can use any DB here, but postgres requires the POSTGRES_USER environment variable
-      - image: postgres:9.4.1
-        environment:
-          POSTGRES_USER: root
+      # In 2.0, you can use multiple different images and group them as a 'Pod'.
+      # Because Docker containers normally don't include DBs, you need to bring
+      # a build image that serves as DB container in your build.
+      - image: <primary-build-image>
+      - image: <db-image-1>
+      - image: <db-image-2>
 
     steps:
       - checkout
 
-      # There’s no inference in 2.0, which means there’s nothing to override
-      # You’ll need to manually install your project’s dependencies
+      # There’s no inference in 2.0, which means there’s nothing to override.
+      # You’ll need to manually install your project’s dependencies.
       # The current timeout for no output is hardcoded to 600 seconds
-      - run: bundle install --path vendor/bundle
+      - run: <dependency-install-command>
 
       # The DB is not automatically created in 2.0, so we manually set up a test DB
-      - run: sudo -u root createuser -h localhost --superuser ubuntu
-      - run: sudo createdb -h localhost test_db
-      - run: bundle exec rake db:create db:schema:load
-
-      # Run any test commands
-      - run: bundle exec rspec
+      - run: <setup-db-command>
 
+      # Again, there is no inference in 2.0, so nothing to override here, either.
+      # Just use any test commands.
+      - run: <test-command>
```

Comment:
I think this is confusing for people since this section will likely have multiple sub-sections, e.g:

```
      - run:
          name: Run tests
          command: lein test
          environment:
            DATABASE_URL: postgres://conductor:@localhost:5432/conductor_test
```

Can we make it clearer how the `run:` step is used?